workspace-manager-clienttests/src/main/java/scripts/testscripts/DataReferenceLifecycle.java

```diff
@@ -63,10 +62,8 @@ private void assertDataReferenceDescription(
     assertThat(
         dataReferenceDescription.getReferenceType(), equalTo(ReferenceTypeEnum.DATA_REPO_SNAPSHOT));
     assertThat(
-        dataReferenceDescription.getReference().getSnapshot(),
-        equalTo(ClientTestUtils.TEST_SNAPSHOT));
+        dataReferenceDescription.getReference().getSnapshot(), equalTo(getDataRepoSnapshotId()));
```

Comment:
`assertThat` is what's widely used in these TestRunner tests. There's probably a reason for that, but I'll follow up separately to confirm.

Reply:
There is a difference between `assertEquals` versus `assertThat`. The former generates an `ComparisonFailure` exception if String comparison fails with an easy to read message. The latter generates an `AssertionError' if the match failed, it has a wider semantics than `ComparisonFailure`. `JUnit` tests more often use `assertEquals` because the IDE will produce a more readable output.